cmake/modules/BuildOpentelemetry.cmake

```diff
@@ -6,7 +6,7 @@ function(target_create _target _lib)
 endfunction()
 
 function(build_opentelemetry)
-  set(opentelemetry_SOURCE_DIR "${PROJECT_SOURCE_DIR}/src/opentelemetry-cpp")
+  set(opentelemetry_SOURCE_DIR "${PROJECT_SOURCE_DIR}/src/jaegertracing/opentelemetry-cpp")
```

Comment:
could you please use `cmake: ` as the prefix of the title of the commit message? also, might want to explain why we'd want to use submodule in place of `git fetch`.